### PR TITLE
src: adjust generation of web pages for nodejs.org

### PR DESCRIPTION
**Pull request by mhdawson**
Adjust the generateion of the summary web page for the
coverage data to fit the nodejs.org look and feel.

Change this file so that it generates format as shown at https://coverage.nodejs.org

### CODE REVIEW

**Comment by mhdawson** [comment]
@addaleax can you review and L.G.T.M or comment as appropriate.  Once this lands then I'll update the nightly job to clone the testing repo instead of yours.

File: coverage/generate-index-html.py [hunk 1]
```diff
@@ -10,35 +10,67 @@
...
+
+
+  <div id="main">
+      <div class="container">
+
+          <h1>Node.js Code Coverage</h1>
+
   <div class="mdl-layout__drawer">
     <nav class="mdl-navigation">
         <a class="mdl-navigation__link" href="https://github.com/nodejs/node">Node.js Core</a>
```
Comment (addaleax) [inline coverage/generate-index-html.py:76]: I think the line after this one (link to `https://github.com/addaleax/node-core-coverage`) can be removed

**Review by addaleax — APPROVED** [review]
LGTM (but only looked at the diff)

File: coverage/generate-index-html.py (hunk 1 above)
Comment (mhdawson) [reply coverage/generate-index-html.py:76]: Thanks will validate and then remove/land if still ok.

**Comment by mhdawson** [comment]
Landed as ec151b9f052552418d4390a992f4006f9687aa17

**Pull request closed without merging**